tailwind.config.js

```diff
@@ -1,15 +1,33 @@
+const plugin = require('tailwindcss/plugin')
+
 module.exports = {
   content: ['./components/**/*.{js,vue}', './directives/**/*.js'],
-  corePlugins: {
-    preflight: false // Enable later when removing demosplan base styles. See https://tailwindcss.com/docs/preflight
-  },
-  plugins: [],
+  important: true, // Utilities should always win https://sebastiandedeyne.com/why-we-use-important-with-tailwind/
+  plugins: [
+    plugin(function({ addUtilities }) {
+      addUtilities({
+        /**
+         * This overrides the Tailwind class of the same name with some
+         * styles to ensure browsers are adding hyphens if appropriate.
+         */
+        '.break-words': {
+          'word-wrap': 'break-word',
+          'word-break': 'break-word',
+          'hyphens': 'auto'
+        },
+      })
+    })
+  ],
   safelist: [
     {
       pattern: /./// Disable purging https://github.com/tailwindlabs/tailwindcss/discussions/6557#discussioncomment-1838214
     }
   ],
   theme: {
-    extend: {}
+    extend: {
+      flexShrink: {
+        2: '2'
```

Comment:
out of curiosity: Why that?

Reply:
out of the box, [tailwind supports flex-shrink 0 or 1](https://tailwindcss.com/docs/flex-shrink). However in one place in demosplan we use flex-shrink: 2, and this is the official way of [extending tailwind utility classes](https://tailwindcss.com/docs/plugins#extending-the-configuration).